SIS_import/sis_set_url.rb

```diff
@@ -529,24 +504,26 @@ def read_argv
 		process_error = "Cannot find logs directory " + outputDirectory
 	else
 		begin
-			outputFile = File.open(outputDirectory + "Canvas_set_url_#{Time.new.strftime("%Y%m%d%H%M%S")}.txt", "w")
+			outputFileName = outputDirectory + "Canvas_set_url_#{Time.new.strftime("%Y%m%d%H%M%S")}.txt"
+			outputFile = File.open(outputFileName, "w")
```

Comment:
What happens if the file can't be opened for write?  Mucked up permissions do happen if, for example, someone accidentally does an install as the wrong user.